relay-event-normalization/src/normalize/mod.rs

```diff
@@ -257,72 +241,35 @@ pub struct ModelCosts {
     /// The version of the model cost struct
     pub version: u16,
 
-    /// The mappings of model ID => cost (used in version 1)
-    #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub costs: Vec<ModelCost>,
-
-    /// The mappings of model ID => cost as a dictionary (version 2)
+    /// The mappings of model ID => cost as a dictionary
     #[serde(default, skip_serializing_if = "HashMap::is_empty")]
     pub models: HashMap<String, ModelCostV2>,
 }
 
 impl ModelCosts {
-    const MAX_SUPPORTED_VERSION: u16 = 2;
+    const SUPPORTED_VERSION: u16 = 2;
 
     /// `true` if the model costs are empty and the version is supported.
     pub fn is_empty(&self) -> bool {
-        (self.costs.is_empty() && self.models.is_empty()) || !self.is_enabled()
+        self.models.is_empty() || !self.is_enabled()
     }
 
     /// `false` if measurement and metrics extraction should be skipped.
     pub fn is_enabled(&self) -> bool {
-        self.version > 0 && self.version <= ModelCosts::MAX_SUPPORTED_VERSION
+        self.version == 2
```

Comment:
This check should probably be the same as on line 264 below, right?
```suggestion
        self.version == Self::SUPPORTED_VERSION
```